src/pe/header.rs

```diff
@@ -569,6 +570,30 @@ impl Header {
             optional_header,
         })
     }
+
+    /// Parses PE header from the given bytes that not contains DOS stub, default DosHeader will be used
+    pub fn parse_without_dos(bytes: &[u8]) -> error::Result<Self> {
+        let dos_header = DosHeader::default();
+        let mut offset = dos_header.pe_pointer as usize;
+        let signature = bytes.gread_with(&mut offset, scroll::LE).map_err(|_| {
+            error::Error::Malformed(format!("cannot parse PE signature (offset {:#x})", offset))
+        })?;
+        let coff_header = CoffHeader::parse(bytes, &mut offset)?;
+        let optional_header = if coff_header.size_of_optional_header > 0 {
+            let opt_hdr = bytes.pread::<optional_header::OptionalHeader>(offset)?;
+            Some(opt_hdr)
+        } else {
+            None
+        };
+
+        Ok(Header {
+            dos_header,
+            dos_stub: DosStub::default(),
+            signature,
+            coff_header,
+            optional_header,
+        })
```

Comment:
I should have said this before, but all of this is basically identical, it might be nice to make a private function:

```rust
fn parse_impl(bytes: &[u8]), header: DosHeader, stub: DosStub) -> Self {
  // highlighted section
}
```
and then the two pub functions call this respectively (your new function supply Default to the header and stub args)